README.adoc

```diff
@@ -80,13 +80,13 @@ The list of available container images is sourced from the cluster's internal co
 
 To list the available components and associated container images for your cluster:
 
-. Log in to the {product-title} cluster with {odo-title}:
+. Access your cluster. Authentication with odo login for Kubernetes clusters is currently not supported. For OpenShift cluster, you can authenticate with `odo login`: 
```

Comment:
Please change it to "`odo login` for Kubernetes..."

Reply:
@boczkowska sorry about the nitpick. But I think `odo login` makes more sense. :)

Reply:
@dharmit done. :relaxed: